backend/Watchdog.Core/Watchdog.Core.Common/Enums/Issues/IssueStatus.cs

```diff
@@ -0,0 +1,9 @@
+namespace Watchdog.Core.Common.Enums.Issues
+{
+    public enum IssueStatus
+    {
+        Active = 0,
```

Comment:
You don't need to write numbers here. They are set by default

Reply:
removed